Project2015To2017/Transforms/FileTransformation.cs

```diff
@@ -156,15 +156,19 @@ private static bool KeepFileInclusion(XElement x)
 				knownFullPaths.Add(Path.GetFullPath(Path.Combine(projectFolder.FullName, otherIncludeMatchingWildcard)));
 			}
 
-			foreach (var nonListedFile in filesInFolder.Except(knownFullPaths, StringComparer.OrdinalIgnoreCase))
+			// if (!project.IsModernProject)
```

Comment:
The only change there is additional indentation level for future `if` statement.